specification/iotoperations/IoTOperations.Management/models/dataflows/dataflowEndpoints.tsp

```diff
@@ -153,8 +137,20 @@ model DataflowEndpointDataLakeStorage {
   batching?: BatchingConfiguration;
 }
 
+@doc("Azure Data Lake endpoint Authentication properties.  NOTE Enum - Only one method is supported for one entry")
+model DataflowEndpointDataLakeStorageAuthentication {
```

Comment:
Shouldn't each of these also have the `AuthenticationMethod` enum?

Reply:
so the update here would be to include `AuthenticationMethod` within each endpoint type?

Reply:
That was what we wanted right? @chgennar and @bilalsellak